### PR TITLE
Update randomgift-sheet.gs

### PR DESCRIPTION
**Pull request by cristobalgh**
Strings translated to Spanish (from Chile =)), prioritizing context meaning, not literal translation.

### CODE REVIEW

File: randomgift-sheet.gs [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
+  "sent": "Felicidades, se ha completado la distribución y todos los correos han sido enviados. Disfruta tu evento!",
+  "error": "Lo siento, algo salió mal."
 }
 
 var msgMailEN = {
-  "dear": "Dear",
-  "participants": "The people participating are",
-  "signature": "Enjoy!",
-  "target": "is your target for this event.",
+  "dear": "Estimad@",
```
Comment (steren) [inline randomgift-sheet.gs:10]: Are you using a "@" in order to stay neutral?
Is this something that is common? what about using "Estimada / Estimado"?

**Comment by steren** [comment]
I added your change in https://github.com/steren/secret-santa/commit/ff595f0e2ecc8881f6a819748a79eb200023e7dc

Let me know if it works correctly.

Note that the UI is not translated, only the messages

File: randomgift-sheet.gs (hunk 1 above)
Comment (cristobalgh) [reply randomgift-sheet.gs:10]: Yes, I'm using @ to stay neutral... It's not a formal use, but it's common and a lot shorter (and friendlier) than Estimado / Estimada...

**Comment by cristobalgh** [comment]
Thanks!,
not sure how to add it to gsheet again.. I used the add on manager...
Tried but didn't see were to choose language or if it detect it
automatically from the sheet settings?

Thanks

**Comment by steren** [comment]
The right panel is not translated (the strings are in [another file](https://github.com/steren/secret-santa/blob/appsscripts/Sidebar.html)), but the emails should be translated automatically based on the language of your account. Can you confirm?

**Comment by cristobalgh** [comment]
I've translated the other file
<https://github.com/steren/secret-santa/blob/appsscripts/Sidebar.html> also.

I changed the language of my account to spanish (used in english), but now
when trying to test it, I get:
[image: image.png]

The test data is just (tried with other data but I get the same error)...

[image: image.png]

Thank you very much,
regards

**Comment by steren** [comment]
Thanks for the translation.
I do not see the pictures in your message. Would you mind describing the issue ?

**Comment by cristobalgh** [comment]
No problem!
It assigns the recipients but doesn't send the emails because:

Sorry, something went wrong:
TypeError: Cannot find function split in object 1.

**Comment by steren** [comment]
Hi,
I cannot reproduce the issue.

Do you confirm that there is no comma in the participant names? 
Can you start from a new sheet?

**Comment by cristobalgh** [comment]
Hi!,
I started on a new sheet and also on a new gmail account.. Still having the
same issue:

Sorry, something went wrong.
TypeError: Cannot find function split in object 7.

(Some times is 7, sometimes other numbers (1, 3, 9, etc)).

http://prntscr.com/m1m3vn ->error

Here is one of the test files:
https://docs.google.com/spreadsheets/d/1_XGJHhh0YtDhdTTiYLP3yPyGTBmUCfHKhIc6AQtO6Ek/edit?usp=sharing

Sorry for all the trouble, before this it worked perfectly.

There are no commas on the test data set, this is it:

http://prntscr.com/m1m425

Only my email is real.

Regards, have nice holidays.

**Comment by cristobalgh** [comment]
Forgot to mention that I'm using the gsheet complement, not this page:

https://www.cadeaux-entre-nous.fr/

**Comment by cristobalgh** [comment]
And last thing!

It assigns the targets, but fails on the sending emails phase...

**Comment by steren** [comment]
I could reproduce the issue, it seems that I broke the add-on for everybody by updating it...

**Comment by steren** [comment]
I had to migrate away from the Chrome Web store (see [blog post](https://cloud.google.com/blog/products/g-suite/migrating-gsuite-extensions-from-chrome-web-store-to-gsuite-marketplace)).

I think I only fixed the version in the GSuite marketplace. 

Let's see after Google completes the marketplace if there is still an issue.

**Comment by cristobalgh** [comment]
Thanks!

Cristóbal Galleguillos Hart
+569 9479 6885

**Pull request closed without merging**